Makefile

```diff
@@ -72,7 +74,7 @@ tests-locally: install
 	. $(VENV)/bin/activate; pytest $(PYTEST_ARGS)
 
 lint-locally: install
-	. $(VENV)/bin/activate; ./scripts/run_lint.sh
```

Comment:
Do we want to remove this file or is it use elsewhere?

Reply:
I would vote for removing it and rely only on one place

Reply:
The lint-errors Makefile target uses it but we could get rid of that target. (The target does what the old CI test did: fail if pylint flags anything as an error)

Reply:
removed in the latest commit